### PR TITLE
Outline for tabbed add as text/user buttons

### PR DESCRIPTION
**Pull request by DoubleCookies**
#857 

Hope I placed CSS for those buttons in correct file (or it is better to create a new one?)
add outline

### CODE REVIEW

**Comment by DoubleCookies** [comment]
Good idea!

**Comment by DoubleCookies** [comment]
It's ready to merge

File: app/routes/calendar/$id/report-winners.tsx [hunk 1]
```diff
@@ -402,6 +402,7 @@ function Players({
                 variant="minimal"
                 onClick={() => handlePlayerInputTypeChange(i)}
                 data-cy="change-input-type-button"
+                className={"outline-theme"}
```
Comment (Sendouc) [inline app/routes/calendar/$id/report-winners.tsx:405]:
```suggestion
                className="outline-theme"
```

This is personal preference.... should lint for this probably :)

**Comment by Sendouc** [comment]
Very nice!

**Pull request merged**